### ISSUE TITLE
Kubeaudit throws errors instead of warning for unsupported types

### ISSUE THREAD

**Issue by alexhope61** [Shopify/kubeaudit#235]
##### ISSUE TYPE
 - [x] Bug Report
 - [ ] Feature Idea

# BUG REPORT

##### SUMMARY
Kubeaudit returns an error instead of a warnig for k8s `Jobs` because they're unsupported.

##### ENVIRONMENT
* Kubeaudit version: latest
* Kubeaudit install method: DIY-BUILD/Github app

##### STEPS TO REPRODUCE

Run `kubeaudit all -f validJobTemplate.yml`

##### EXPECTED RESULTS

Receive a warning that Jobs are not supported by kubeaudit

##### ACTUAL RESULTS

Kubeaudit threw an error that caused my CI to fail

##### ADDITIONAL INFORMATION

![image](https://user-images.githubusercontent.com/22034090/66661011-38ebf680-ec14-11e9-9985-175b51bb4c25.png)

**Comment by mike-stewart** [comment]
I get a similar error:
```
ERRO[0000] getResources failed                          
ERRO[0000] File is not a valid Kubernetes manifest: no kind "APIService" is registered for version "apiregistration.k8s.io/v1beta1" in scheme "pkg/runtime/scheme.go:101" 
```

Anything I can do to help move towards a fix? In terms of repro steps or potentially attempting a fix if you can point me in the right direction (haven't spent much time with go). @klautcomputing @nschhina genevieveluyt

**Comment by nschhina** [comment]
Need to include schema for CustomDefinedResources, if such a schema doesn't exist- you'll have to write an entire schema for the same and then import it in `pkg/runtime/scheme.go`

**Comment by mike-stewart** [comment]
@nschhina Rather than defining a schema for every custom resource, is there some way we can get kubeaudit to ignore custom resources?

**Comment by genevieveluyt** [comment]
Hey @mike-stewart, thanks for offering to help! I'm actually in the middle of redesigning kubeaudit (see https://github.com/Shopify/kubeaudit/pull/234) after which it will only have warnings for any resource it doesn't recognize. That will hopefully be done in the coming weeks, but if this is blocking you, I can do a quick temporary patch this week.

**Comment by mike-stewart** [comment]
@genevieveluyt Awesome! It will block adoption for us, but we can wait a few weeks. Let me know if you need a beta tester.

**Comment by genevieveluyt** [comment]
@mike-stewart having a beta tester would be great! I will let you know when it's ready, thank you!

**Comment by genevieveluyt** [comment]
This should be fixed by #243. Please re-open if the problem remains
